mixedLineEnding.fsx

```diff
@@ -0,0 +1,32 @@
+open System.IO
+open System
+
+let mixedLineEnding(fileInfo: FileInfo) = 
```

Comment:
@realmarv we use PascalCase for functions

Reply:
Done